components/builder-web/app/origin/origin-page/origin-job-detail/origin-job-detail.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="origin-job-detail-component body">
+  <div class="content">
+    <h3>
+      <a (click)="backToGroups()">
+        Build Job Groups
+      </a>&nbsp;/ Group {{ this.id }}
+    </h3>
+    <section class="jobgroup-overview">
+      <table class="jobgroup-table">

```

Comment:
I'd rather this not be `table`-based. Let's handle this mobile-first with stacked `div`s and scale up on the desktop with Flexbox. Liz is working on some designs, I believe.

Reply:
That could be done, but this is a literal table of data, even if it only has one row. Seems a perfectly appropriate place for a table?

Reply:
I disagree; there are no rows of data, just labels and values. The mobile view makes this a bit clearer.